.github/workflows/create-design-review-issues.yml

```diff
@@ -0,0 +1,32 @@
+name: Create Design Review Issues
+
+on:
+  schedule:
+    # Every Wednesday at 00:00:00
+    - cron: '0 0 * * 3'
+
+jobs:
+  sweep-experiments:
+    name: Create Design Review Issues
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Up Node
+        uses: actions/setup-node@v2.1.5
+
+      - name: Generate
+        id: generate
+        run: |
+          issue=$(wget -q -O - https://raw.githubusercontent.com/ampproject/amphtml/master/build-system/tasks/create-design-review-issues/index.js | node)
+          issue="${issue//'%'/'%25'}"
+          issue="${issue//$'\n'/'%0A'}"
+          issue="${issue//$'\r'/'%0D'}"
+          echo "::set-output name=issue::$(echo "$issue")"
+
+      - name: Create Issue
+        uses: nashmaniac/create-issue-action@v1.1
```

Comment:
just [checked out the repo](https://github.com/nashmaniac/create-issue-action) and it only has 1 star / very little activity. are you tied to this action?

it looks like there is a more popular one at [JasonEtco/create-an-issue](https://github.com/JasonEtco/create-an-issue)

Reply:
Given how little code is present in these actions, might consider rolling your own for this.

But +1 to the suggestion here.

Reply:
I just did what was easiest. But we can do a simple POST request instead, so rolling my own now. Bonus that the task time reduced from ~40s to ~4s.